### DISCUSSION TITLE
redirect_uri with multiple callback URLs not working for GitHub app

### DISCUSSION THREAD

**Discussion by rohanmayya** [community/community#103058] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

I'm trying to set up multiple callback URLs for my GitHub app that can be installed to use for any of our users.

The problem is the disparity in local and production testing. I've tried to specify multiple callback URLs and conditionally set the redirect_uri based on the environment, but it always redirects me back to production after the setup is done.

For reference, my redirect_uris are:
http://localhost:3000/github/success
https://xyz.com/github/success

And when the user clicks on the installations link, I DO specify the redirect_uri, but it still does not work.

May I please know how to fix this?

**Comment by eleftherias** [comment]
This looks like the same issue described in this topic https://github.com/orgs/community/discussions/64705
